mava/advanced_usage/ff_ippo_store_experience.py

```diff
@@ -687,19 +685,19 @@ def _reshape_experience(experience: Dict[str, chex.Array]) -> Dict[str, chex.Arr
             lambda x: x[:, 0, ...],
             learner_output.learner_state.params.actor_params,  # Select only actor params
         )
-        rng_e, *eval_rngs = jax.random.split(rng_e, n_devices + 1)
-        eval_rngs = jnp.stack(eval_rngs)
-        eval_rngs = eval_rngs.reshape(n_devices, -1)
+        key_e, *eval_keys = jax.random.split(key_e, n_devices + 1)
+        eval_keys = jnp.stack(eval_keys)
+        eval_keys = eval_keys.reshape(n_devices, -1)
 
         # Evaluate.
-        evaluator_output = evaluator(trained_params, eval_rngs)
+        evaluator_output = evaluator(trained_params, eval_keys)
         jax.block_until_ready(evaluator_output)
 
         # Log the results of the evaluation.
         elapsed_time = time.time() - start_time
         evaluator_output.episodes_info["steps_per_second"] = steps_per_rollout / elapsed_time
-        episode_return = log(
-            metrics=evaluator_output,
+        episode_return = logger.log_evaluator_metrics(
```

Comment:
Minor suggestion: If we agree that eval stands for things related to the evaluation phase can we shrink this a bit to `logger.log_eval_metrics`?
But if you feel that way is more informative keep it 😅

Reply:
I think this is very trivial I am happy either way hh